app/_src/guides/customizing-transparent-proxy-configuration-in-kubernetes-mode.md

```diff
@@ -0,0 +1,434 @@
+
+{% warning %}
+The ConfigMap in the `{{ kuma-system }}` namespace is required for proper operation, so it must be present even if custom ones are used in all individual workload namespaces.
+{% endwarning %}
+
+### Custom ConfigMap Name
+
+The name of the ConfigMap is defined by the `{{ transparentProxy }}.configMap.name` setting. By default, this name is set to `kuma-transparent-proxy-config`.
+
+To apply a custom name for all workloads, you can modify the `{{ transparentProxy }}.configMap.name` setting during installation:
+
+{% cpinstall transparent-proxy-config-configmap-custom-name-globally %}
+{{ transparentProxy }}.configMap.name=custom-name
+{% endcpinstall %}
+
+To use a different resource for specific workloads, apply the `traffic.kuma.io/transparent-proxy-configmap-name` annotation to those workloads. For example:
+
+```sh
+kubectl annotate pods <pod_name> "traffic.kuma.io/transparent-proxy-configmap-name=custom-name"
+```
+
+This allows flexibility, enabling different workloads to use distinct ConfigMap settings while maintaining a global default.
+{% endif_version %}
+
+## Method {% inc method_number %}: Kubernetes Annotations
+{:#{{ tproxy.ids.guides.customize-config.k8s.methods.annotations }}}
+
+Kubernetes annotations can be applied to individual workloads to modify the transparent proxy configuration. These annotations allow fine-tuning of specific behaviors for a single workload without affecting others.
+
+{% if_version gte:2.9.x %}
+{% tip %}
+Before {{ Kuma }} 2.9, annotations were the only way to modify the transparent proxy configuration on a per-workload basis. In {{ Kuma }} 2.9, the [{{ configuration-in-configmap }}](#{{ tproxy.ids.guides.customize-config.k8s.methods.configmap }}) feature was introduced, offering a more efficient way to configure groups of workloads. When this feature is no longer experimental, it will become the recommended approach for managing transparent proxy settings across multiple workloads. Annotations will still be useful for fine-tuning specific configurations when you need an individual workload to behave differently from the others.
+{% endtip %}
+{% endif_version %}
+
+Below you can find a list of annotations that can be used to configure specific transparent proxy settings. 
+
+- `traffic.kuma.io/exclude-inbound-ips`
+
+- `traffic.kuma.io/exclude-inbound-ports`
+
+- `traffic.kuma.io/exclude-outbound-ips`
+
+- `traffic.kuma.io/exclude-outbound-ports`
+
+- `traffic.kuma.io/exclude-outbound-ports-for-uids`
+
+- `traffic.kuma.io/drop-invalid-packets`
+
+- `traffic.kuma.io/iptables-logs`
+
+- `kuma.io/transparent-proxying-ebpf`
+
+- `kuma.io/transparent-proxying-ebpf-bpf-fs-path`
+
+- `kuma.io/transparent-proxying-ebpf-cgroup-path`
+
+- `kuma.io/transparent-proxying-ebpf-tc-attach-iface`
+
+- `kuma.io/transparent-proxying-ebpf-instance-ip-env-var-name`
+
+- `kuma.io/transparent-proxying-ebpf-programs-source-path`
```

Comment:
this will eventually get out of date, should we just link to the sections in https://kuma.io/docs/2.9.x/reference/kubernetes-annotations ?

Reply:
I don't think these will go out of date to be honest, and it's hard to reference some subsets and it would involve restructuring the reference page which I don't want to do as this PR is already quite big.